### PR TITLE
Change checks for write_page_segmentation

### PR DESCRIPTION
**Pull request by zuphilip**
This seems to solve #88 and allows to run page segmentation also in smaller images (less than 600x600), by using the `-n` option which deactivate the checks. The function `write_page_segmentation` should IMO not depend on the size of the image. Moreover, the check is now equal to the one for the function `read_page_segmentation` (cf. some lines above).

### CODE REVIEW

File: ocrolib/common.py [hunk 1]
```diff
@@ -279,7 +279,7 @@ def read_page_segmentation(fname):
     segmentation = make_seg_black(segmentation)
     return segmentation
 
-@checks(str,PAGESEG)
+@checks(str,_=PAGESEG)
```
Comment (zuphilip) [inline ocrolib/common.py]: What is the meaning of this change? Help...

Comment (kba) [reply ocrolib/common.py]: checks is here https://github.com/tmbdev/ocropy/blob/ace19d5801c2dcd4d669ad4fac53b495e5077fe8/ocrolib/toplevel.py#L195

It expects positional arguments and keyword arguments that map argument names to types which are checked (I think). It can also check the **return** type agains the keyword argument `_` (https://github.com/tmbdev/ocropy/blob/ace19d5801c2dcd4d669ad4fac53b495e5077fe8/ocrolib/toplevel.py#L218).

So your change makes a check that `write_page_segmentation` returns `PAGESEG` (https://github.com/tmbdev/ocropy/blob/ace19d5801c2dcd4d669ad4fac53b495e5077fe8/ocrolib/toplevel.py#L428). Which it does not, it retuns `None`.

In short, I think you could just omit the `checks` decorator :whale:

Comment (zuphilip) [reply ocrolib/common.py]: I don't follow here. AFAIK the change I make is deactivating the check for PAGESEG which is kind of conform to the other deactivated checks at this point. Thus, I think it should be okay to merge this PR. Or do you suggest to delete this line completely?

Comment (kba) [reply ocrolib/common.py]: I think the check does nothing so I would remove it.

Comment (zuphilip) [reply ocrolib/common.py]: Okay, fine for me. There might be more of this sort of checks...

Comment (kba) [reply ocrolib/common.py]: Yeah, it's not ideal because the original check did make sense (checking that the second argument, `image` is one of `PAGESEG`.
